compiler/codegen/OMRCodeGenPhase.cpp

```diff
@@ -180,48 +180,44 @@ OMR::CodeGenPhase::performProcessRelocationsPhase(TR::CodeGenerator * cg, TR::Co
      }
 
    if (cg->getAheadOfTimeCompile() && (comp->getOption(TR_TraceRelocatableDataCG) || comp->getOption(TR_TraceRelocatableDataDetailsCG) || comp->getOption(TR_TraceReloCG)))
-     {
-     traceMsg(comp, "\n<relocatableDataCG>\n");
-     if (comp->getOption(TR_TraceRelocatableDataDetailsCG)) // verbose output
-        {
-        uint8_t * relocatableMethodCodeStart = (uint8_t *)comp->getRelocatableMethodCodeStart();
-        traceMsg(comp, "Code start = %8x, Method start pc = %x, Method start pc offset = 0x%x\n", relocatableMethodCodeStart, cg->getCodeStart(), cg->getCodeStart() - relocatableMethodCodeStart);
-        }
-     cg->getAheadOfTimeCompile()->dumpRelocationData();
-     traceMsg(comp, "</relocatableDataCG>\n");
-     }
+      {
+      traceMsg(comp, "\n<relocatableDataCG>\n");
+      if (comp->getOption(TR_TraceRelocatableDataDetailsCG)) // verbose output
+         {
+         uint8_t * relocatableMethodCodeStart = (uint8_t *)comp->getRelocatableMethodCodeStart();
+         traceMsg(comp, "Code start = %8x, Method start pc = %x, Method start pc offset = 0x%x\n", relocatableMethodCodeStart, cg->getCodeStart(), cg->getCodeStart() - relocatableMethodCodeStart);
+         }
+      cg->getAheadOfTimeCompile()->dumpRelocationData();
+      traceMsg(comp, "</relocatableDataCG>\n");
+      }
```

Comment:
any chance you could split these changes into a separate commit? they interfere with appreciating the setup you've made here for a much better solution to the FD issue.

Reply:
The reason this formatting change was made is because the indentation in it's current state is very confusing [1], and at first sight one may thing the:

```
     if (comp->getCurrentMethod() == NULL)
        {
        comp->getMethodSymbol()->setMethodAddress(cg->getBinaryBufferStart());
        }
```

piece of code is guarded by the `if (cg->getAheadOfTimeCompile() &&` check, which is what I first thought, but upon a more careful inspection, the indentation was hiding this fact.

If you wish to view that commit without whitespace changes GitHub can help here:
https://github.com/eclipse/omr/pull/4907/commits/ab6d6a497983599844cce11f5c1c870e9b160d8c?w=1

[1] https://github.com/eclipse/omr/blob/31db81f3d84a431736ac7a9ff1b3980158fae258/compiler/codegen/OMRCodeGenPhase.cpp#L182-L202

Reply:
On second thought, I think I agree. For history of commits it would be beneficial to split up formatting from actual changes.

I've split up the actual change into 6e902d5, where it belongs, and the formatting change into 
27349e8. Hope that helps!